src/models/get_list.py

```diff
@@ -0,0 +1,54 @@
+with open('domain_raw.txt') as f:
```

Comment:
when you have python scripts like this, its worth considering having a doc string at the beggining describeing what the script does and what it requires. I.e.

```
"""
get_list will retrieve a list and do something from the file ./domain_raw.txt and do another thing.
then it outputs to ./some_file.txt .
"""
with open('domain_raw.txt') as f:
...
```